packages/react/src/InlineLink/InlineLink.module.css

```diff
@@ -13,7 +13,9 @@
 
 .InlineLink,
 .InlineLink > span {
+  font-size: inherit;
```

Comment:
`InlineLink`s will inherit their font family by default. This regression happens because InlineLinks no longer have an inner `<Text>` tag as part of their implementation. In this story, it is a bare `<InlineLink>`, not wrapped by a `<Text>` tag, so fallback to the default system font.

I don't want to add `font-family: var(--brand-body-fontFamily);`, because then font-family will not not be inherited.

I could wrap the Story in a <Text> tag, but then I think we lose the spirit of what the default story is doing (just showing the bare component).

Reply:
Sounds good, thanks for clarifying.